lib/tasks/colligo.rake

```diff
@@ -6,7 +6,9 @@ namespace :colligo do
     Rake::Task['db:migrate'].invoke
     Rake::Task['colligo:download_and_unzip_jetty'].invoke
     Rake::Task['colligo:copy_solr_configs'].invoke
-    Rake::Task['colligo:fixtures'].invoke
+    Jettywrapper.wrap(Jettywrapper.load_config) do
```

Comment:
Yeah, we definitely discussed getting dependencies up to our latest standards.  This is just trying to fix the fact that the install task throws an error.